shuttle/tests/basic/dfs.rs

```diff
@@ -45,86 +45,32 @@ fn trivial_two_threads() {
         });
     }
 
-    assert_eq!(iterations.load(Ordering::SeqCst), 2);
+    assert_eq!(iterations.load(Ordering::SeqCst), 3);
 }
 
 /// We have two threads T0 and T1 with the following lifecycle (with letter denoting each step):
-/// * T0: spawns T1 (S), acquires lock (A), releases lock (R), finishes (F)
-/// * T1:                acquires lock (a), releases lock (r), finishes (f)
+/// * T0: (i) initializes, spawns T1 (s), acquires lock (a), releases lock (r), finishes (e)
```

Comment:
nit: to be consistent, letter should come after description in all cases

`T0: initializes (i), spawns T1 (s), ..`